skills/cypilot/scripts/cypilot/utils/ui.py

```
@@ -0,0 +1,269 @@
+
+
+def file_action(path: str, action: str) -> None:
+    """Print a file action (created/updated/unchanged)."""
+    if _json_mode:
+        return
+    icons = {
+        "created": _c(_GREEN, "+"),
+        "updated": _c(_YELLOW, "~"),
+        "unchanged": _c(_DIM, "="),
+        "skipped": _c(_DIM, "-"),
+        "deleted": _c(_RED, "×"),
```

Ruff (0.15.2)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)